src/main/java/gift/main/dto/KakaoTokenResponse.java

```diff
@@ -0,0 +1,12 @@
+package gift.main.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public record KakaoTokenResponse(
+        @JsonProperty("access_token") String accessToken,        //사용자 액세스 토큰 값
+        @JsonProperty("token_type") String tokenType,          //토큰타입, bearer로 고정
+        @JsonProperty("refresh_token") String refreshToken,       //사용자 리프레시 토큰 값
+        @JsonProperty("expires_in") int expiresIn,             //액세스 토큰과 ID 토큰의 만료 시간(초)
+        @JsonProperty("refresh_token_expires_in") int refreshTokenExpiresIn  //리프레시 토큰 만료 시간(초)
+) {
+}
```

Comment:
모두 스네이크 케이스로 변환해야한다면 , JsonProperty를 각각 달아주는 것 대신 `@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)`를 사용해볼 수 있을 것 같아요.
```suggestion
@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
public record KakaoTokenResponse(
        String accessToken,        //사용자 액세스 토큰 값
        String tokenType,          //토큰타입, bearer로 고정
        String refreshToken,       //사용자 리프레시 토큰 값
        int expiresIn,             //액세스 토큰과 ID 토큰의 만료 시간(초)
        int refreshTokenExpiresIn  //리프레시 토큰 만료 시간(초)
) {
}
```

Reply:
헉 이 부분 진짜 몰랐습니다...!!
이런식으로 코드를 많이 짰었는데 처음알았습니다. 코드가 더 깔끔해지겠군요🔥🔥🔥 
반영하겠습니다 감사합니다.

Reply:
반영했습니다!